### ISSUE TITLE
More documentation and comments planned?

### ISSUE THREAD

**Issue by MarcoMeter** [kakaobrain/brain_agent#2]
Hello everyone!

Thanks for your promising project!

Are you guys planning on adding further documentation like how to utilize other environments?
DMLab is only available for Linux. Does your code run on Windows or Mac as well?

Also, are you planning on commenting your code to make it more readable? This would be really helpful to actually work with it.

**Comment by seungeunrho** [comment]
Hi! Thanks for your comments.
Yes we plan to add some documentations and comments, but it will take a week or two.

**Comment by MarcoMeter** [comment]
I'm excited to hear that!

I'm currently trying to get a hold of everything using the debugger which is pretty cumbersome due to the asynch processes.
I came across one thing, which confuses me:
During the forward pass triggered by the PolicyWorker the qlen is `1`, while the LearnerWorker causes a qlen of `96`.
Doesn't this mean that the qlen has much more information during optimization?

I used the default.yaml, but made those changes for debugging:
```
num_workers: 1
num_envs_per_worker: 1
num_splits: 1
set_workers_cpu_affinity: False
```
